### ISSUE TITLE
Write local introductions for Guilford County

### ISSUE THREAD

**Issue by ejaxon** [CodeForNC/reentry-resources-hub#23]
## This is a: 

* [ ] bug
* [x] enhancement
* [ ] feature-discussion

# Enhancement
## Description
We need Guilford-county-specific introductions written for each of the pages/topic areas below. 

* [ ] Home
* [ ] Housing
* [ ] Jobs
* [ ] Public Benefits
* [ ] Health Care
* [ ] Education
* [ ] Legal
* [ ] Support Programs
* [ ] Other Resources

If you take on a topic, let @ejaxon or @marshallj know in the comments below and we will mark it as in progress.

## Implementation
You can get an idea of what's required from the corresponding pages of the original [Buncombe County site](http://www.buncombereentryhub.org/) since that information has been reasonably well vetted by subject-area experts.

The content will need to be in HTML and added to this repository, but if you are not comfortable doing that, feel free to compose in a text editor, Word, or Google Docs and then work with a developer to translate into HTML and merge into the code. 

The content for a given topic X goes in the corresponding ````content/jurisdictions/guilford/X/description.txt```` file.

**Comment by marshallj** [comment]
@Sadaajit and I will begin the legwork on this. We may not be doing the writing ourselves. but we will be coordinating with the subject matter experts to collect the info. We have been invited to a meeting on January 23rd of Greensboro Community Stakeholders doing reentry work to briefly talk about the project and begin making local connections that will provide content.

**Comment by marshallj** [comment]
@ejaxon Could you clarify this:

"We need Guilford-county-specific introductions written for each of the pages/topic areas below. These introductions should be valid for ALL counties in the state, i.e., they should not include information specific to any particular county."

Should these intros be specific to Guilford or apply to all counties?

**Comment by ejaxon** [comment]
These are the intro parts specifically relevant to Guilford (the common intros are in [this issue](https://github.com/CodeForNC/reentry-resources-hub/issues/16), which @sparklebunny was starting to work on. I also created a task specifically for [Buncombe County](https://github.com/CodeForNC/reentry-resources-hub/issues/22). Other counties for now will just have the common introductions and the common and NC211-localized resources.

BTW I also created issues for common and Buncombe County highlighted resources - feel free to create one for Guilford (as well as regular resources, which I didn't get to).

Incidentally, we'll have to see how well that page information design works in practice - let me know if we need to rethink it.
